src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,14 +1,96 @@
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.RemoveEventHandler(Microsoft.CodeAnalysis.SyntaxNode event, Microsoft.CodeAnalysis.SyntaxNode handler) -> Microsoft.CodeAnalysis.SyntaxNode
+abstract Microsoft.CodeAnalysis.Options.OptionSet.GetOption(Microsoft.CodeAnalysis.Options.OptionKey optionKey) -> object (forwarded, contained in Microsoft.CodeAnalysis)
+abstract Microsoft.CodeAnalysis.Options.OptionSet.WithChangedOption(Microsoft.CodeAnalysis.Options.OptionKey optionAndLanguage, object value) -> Microsoft.CodeAnalysis.Options.OptionSet (forwarded, contained in Microsoft.CodeAnalysis)
 Microsoft.CodeAnalysis.ApplyChangesKind.ChangeDocumentInfo = 16 -> Microsoft.CodeAnalysis.ApplyChangesKind
 Microsoft.CodeAnalysis.Document.WithFilePath(string filePath) -> Microsoft.CodeAnalysis.Document
 Microsoft.CodeAnalysis.Document.WithFolders(System.Collections.Generic.IEnumerable<string> folders) -> Microsoft.CodeAnalysis.Document
 Microsoft.CodeAnalysis.Document.WithName(string name) -> Microsoft.CodeAnalysis.Document
 Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TypeExpression(Microsoft.CodeAnalysis.ITypeSymbol typeSymbol, bool addImport) -> Microsoft.CodeAnalysis.SyntaxNode
+Microsoft.CodeAnalysis.Options.IOption (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.DefaultValue { get; } -> object (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.DefaultValue.get -> object (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Feature { get; } -> string (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Feature.get -> string (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.IsPerLanguage { get; } -> bool (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.IsPerLanguage.get -> bool (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Name { get; } -> string (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Name.get -> string (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.StorageLocations { get; } -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Options.OptionStorageLocation> (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.StorageLocations.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Options.OptionStorageLocation> (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Type { get; } -> System.Type (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.IOption.Type.get -> System.Type (forwarded, contained in Microsoft.CodeAnalysis)
+Microsoft.CodeAnalysis.Options.Option<> (forwarded, contained in Microsoft.CodeAnalysis)
```

Comment:
dotnet/roslyn-analyzers#1192 filed to track the mishandling of the generic type here.